spec/factories/agencies.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :agency do
+    name { 'Agency name' }
```

Comment:
I'd love to see a couple of semantic options here, see `spec/factories/businesses.rb#5` - those semantic options go in en.yml under the Faker name key, with their own custom key.

Here's a list of suggestions:
```
Children's Aid and Family Services
Community Child Care Connection
Maui Family Support Services
Wisconsin Shares
Child Care Aware of America
```
```suggestion
    name { Faker::Name.agencies }
```